.isort.cfg

```diff
@@ -0,0 +1,12 @@
+[settings]
+indent=4
+combine_star=1
+combine_as_imports=1
+include_trailing_comma=1
+multi_line_output=3
+line_length=88
+lines_after_imports=2
+known_pytest=pytest
+default_section=THIRDPARTY
+sections=FUTURE,STDLIB,PYTEST,DJANGO,THIRDPARTY,FIRSTPARTY,LOCALFOLDER
+known_third_party = base,build_release,check_release,commit_release,create_github_release,deploy_release,prepare_release,publish_release,tomlkit
```

Comment:
In the interest of consolidating configuration files, I prefer that (when possible) we put all of these kinds of settings in `pyproject.toml`. You can see an example of this here: https://github.com/justinmayer/kagi/blob/master/pyproject.toml#L59-L77

Reply:
agree, think I did this before knowing that I can include in pyproject, as now I always put everything I could in there...